status/views.py

```diff
@@ -0,0 +1,107 @@
+"""
+Status checks for LORE.
+
+Notes:
+
+* Useful messages are logged, but "no config found" is returned whether
+  settings are missing or invalid, to prevent information leakage.
+* Different services provide different information, but all should return
+  "up," "down", or "no config found" for the "status" key.
+"""
+
+import re
+from datetime import datetime
+import logging
+
+from django.conf import settings
+from django.http import JsonResponse
+from elasticsearch import Elasticsearch, ConnectionError as ESConnectionError
+from psycopg2 import connect, OperationalError
+from redis import StrictRedis, ConnectionError as RedisConnectionError
+
+log = logging.getLogger(__name__)
+
+
+def get_pg_info():
+    """Check PostgreSQL connection."""
+    try:
+        conf = settings.DATABASES['default']
+    except (AttributeError, KeyError):
```

Comment:
As discussed in meatspace, we're going to return a [503](http://www.checkupdown.com/status/E503.html) status code and the JSON.